documentation/src/main/wiki/RestAPI.md

```diff
@@ -16,90 +16,90 @@ Base api path:
 * **URL**: `/api/artifact`
 * **HTTP Method**: GET
 * **Parameters**: `company`, `project`, `id`
-* **Example**: http://aet.example.com/api/artifact?company=cognifide&project=example&id=56fa80c1ab21c61f14bfef45
+* **Example**: `http://aet.example.com:8181/api/artifact?company=cognifide&project=example&id=56fa80c1ab21c61f14bfef45`
```

Comment:
Ports here are unnecessary, this is example. In the production-like environment, api will be exposed via some proxy like Apache or Ngnix.

Reply:
Ok, ports no. deleted in all example urls